vector/src/main/java/im/vector/app/features/home/room/list/RoomListViewModel.kt

```diff
@@ -145,45 +145,18 @@ class RoomListViewModel @AssistedInject constructor(
 
     companion object : MavericksViewModelFactory<RoomListViewModel, RoomListViewState> by hiltMavericksViewModelFactory()
 
-<<<<<<< HEAD
-    private val roomListSectionBuilder = if (appStateHandler.getCurrentRoomGroupingMethod() is RoomGroupingMethod.BySpace) {
-        RoomListSectionBuilderSpace(
-                session,
-                stringProvider,
-                appStateHandler,
-                viewModelScope,
-                autoAcceptInvites,
-                {
-                    updatableQueries.add(it)
-                },
-                suggestedRoomJoiningState,
-                !vectorPreferences.prefSpacesShowAllRoomInHome()
-        )
-    } else {
-        RoomListSectionBuilderGroup(
-                viewModelScope,
-                session,
-                stringProvider,
-                appStateHandler,
-                autoAcceptInvites
-        ) {
-            updatableQueries.add(it)
-        }
-    }
-=======
     private val roomListSectionBuilder = RoomListSectionBuilder(
             session,
             stringProvider,
             appStateHandler,
             viewModelScope,
             autoAcceptInvites,
             {
-                updatableQuery = it
+                updatableQueries.add(it)
```

Comment:
@yostyle I'm not sure the list of queries is still necessary, it was added to handle the room filtering in the current screen, but it should not be necessary anymore since we open a new screen like in Element, right?

Reply:
You right

Reply:
I will do it in a dedicated commit after the rebase

Reply:
https://github.com/tchapgouv/tchap-android/pull/712/commits/501a91e938a5d76bbe01f64d8511fa5713baf660